packages/flutter_driver/test/src/timeline_summary_test.dart

```diff
@@ -20,11 +20,16 @@ void main() {
       }));
     }
 
-    Map<String, dynamic> build(int timeStamp, int duration) => <String, dynamic>{
+    Map<String, dynamic> frameBegin(int timeStamp) => <String, dynamic>{
```

Comment:
Nit: it seems that we don't have to define `frameBegin` and `frameEnd` separately. We can just reuse the `build` signature, and does the work of `frameBegin` and `frameEnd` inside. That removes the possibility that someone could call multiple `frameBegin` and forget to call any `frameEnd` during a test.

Reply:
this test is used to verify that the data can be obtained correctly when the `frameBegin` and `frameEnd` are not matched.  So add test('skips leading "end" events'）and test('skips leading "end" events'), keeping the same with the `begin` and `end`.